### ISSUE TITLE
ReferenceError: ResizeObserver is not defined: Something went wrong

### ISSUE THREAD

**Issue by jeevadotnet** [wazuh/wazuh-docker#757]
Wazuh Version: `4.3.10`
Docker Version: `20.10.17`
lsb-release -a: `Ubuntu 20.04.4 LTS`
Edition: `Community`
Mode: `Single-node`

Note: Testing Wazuh with a stock standard version, only difference is, is that I changed the dashboard & manager port to `5601:5601` and `55001:55000`.

I've deployed one agent to test if it connects to the server, which it does.
![image](https://user-images.githubusercontent.com/12102129/204812622-9868aafd-4fdf-4cee-bdf6-8ca5d7098ba4.png)

However when I click on the `1` of `total agents / active agents`, i get the following: `Something went wrong`
![image](https://user-images.githubusercontent.com/12102129/204812737-3c6bb286-71c2-40b1-b5f1-2092a1993f91.png)

Details:
```
ReferenceError: ResizeObserver is not defined

    in ChartDonut
    in div
    in u
    in div
    in EuiFlexGroup
    in div
    in VisualizationBasic
    in div
    in u
    in div
    in EuiFlexGroup
    in div
    in div
    in Unknown
    in div
    in u
    in div
    in EuiFlexGroup
    in div
    in u
    in div
    in sy
    in AgentsPreview
    in Unknown
    in Unknown
    in Unknown
    in Provider
    in WzReduxProvider
    in withErrorBoundary(withErrorBoundary(withErrorBoundary(withErrorBoundary(withErrorBoundary(withErrorBoundary(withErrorBoundary(withErrorBoundary(withErrorBoundary(withErrorBoundary(withErrorBoundary(withErrorBoundary(UnknownComponent))))))))))))
    in ErrorBoundary
    in Unknown
```

**Comment by jeevadotnet** [comment]
![image](https://user-images.githubusercontent.com/12102129/205008404-6b5215ab-2533-456b-a973-ff35ee98660f.png)

**Comment by teddytpc1** [comment]
Hi @jeevadotnet.
I was not able to reproduce the issue. It seems to be a new installation. Can you remove the docker deployment, remove the volumes and deploy Wazuh again? 
After that, try to connect the agent again and then reproduce the issue. 
Let me know if it persists, please.

**Comment by jeevadotnet** [comment]
@teddytpc1 

I did it again as requested.
stopped all the containers, `rm` all the containers, `rmi` all the images
removed all volumes `docker volume rm $(docker volume ls -q)`
delete the github clone 
stopped the wazuh-agent on my test VM

deployed as per the instructions, only modifying my 2 ports. 

Still get the same `ReferenceError: ResizeObserver is not defined` 

```
# Wazuh App Copyright (C) 2017, Wazuh Inc. (License GPLv2)
version: '3.7'

services:
  wazuh.manager:
    image: wazuh/wazuh-manager:4.3.10
    hostname: wazuh.manager
    restart: always
    ports:
      - "1514:1514"
      - "1515:1515"
      - "514:514/udp"
      - "55001:55000"
    environment:
      - INDEXER_URL=https://wazuh.indexer:9200
      - INDEXER_USERNAME=admin
      - INDEXER_PASSWORD=SecretPassword
      - FILEBEAT_SSL_VERIFICATION_MODE=full
      - SSL_CERTIFICATE_AUTHORITIES=/etc/ssl/root-ca.pem
      - SSL_CERTIFICATE=/etc/ssl/filebeat.pem
      - SSL_KEY=/etc/ssl/filebeat.key
      - API_USERNAME=wazuh-wui
      - API_PASSWORD=MyS3cr37P450r.*-
    volumes:
      - wazuh_api_configuration:/var/ossec/api/configuration
      - wazuh_etc:/var/ossec/etc
      - wazuh_logs:/var/ossec/logs
      - wazuh_queue:/var/ossec/queue
      - wazuh_var_multigroups:/var/ossec/var/multigroups
      - wazuh_integrations:/var/ossec/integrations
      - wazuh_active_response:/var/ossec/active-response/bin
      - wazuh_agentless:/var/ossec/agentless
      - wazuh_wodles:/var/ossec/wodles
      - filebeat_etc:/etc/filebeat
      - filebeat_var:/var/lib/filebeat
      - ./config/wazuh_indexer_ssl_certs/root-ca-manager.pem:/etc/ssl/root-ca.pem
      - ./config/wazuh_indexer_ssl_certs/wazuh.manager.pem:/etc/ssl/filebeat.pem
      - ./config/wazuh_indexer_ssl_certs/wazuh.manager-key.pem:/etc/ssl/filebeat.key
      - ./config/wazuh_cluster/wazuh_manager.conf:/wazuh-config-mount/etc/ossec.conf

  wazuh.indexer:
    image: wazuh/wazuh-indexer:4.3.10
    hostname: wazuh.indexer
    restart: always
    ports:
      - "9200:9200"
    environment:
      - "OPENSEARCH_JAVA_OPTS=-Xms512m -Xmx512m"
    ulimits:
      memlock:
        soft: -1
        hard: -1
      nofile:
        soft: 65536
        hard: 65536
    volumes:
      - wazuh-indexer-data:/var/lib/wazuh-indexer
      - ./config/wazuh_indexer_ssl_certs/root-ca.pem:/usr/share/wazuh-indexer/config/certs/root-ca.pem
      - ./config/wazuh_indexer_ssl_certs/wazuh.indexer-key.pem:/usr/share/wazuh-indexer/config/certs/wazuh.indexer.key
      - ./config/wazuh_indexer_ssl_certs/wazuh.indexer.pem:/usr/share/wazuh-indexer/config/certs/wazuh.indexer.pem
      - ./config/wazuh_indexer_ssl_certs/admin.pem:/usr/share/wazuh-indexer/config/certs/admin.pem
      - ./config/wazuh_indexer_ssl_certs/admin-key.pem:/usr/share/wazuh-indexer/config/certs/admin-key.pem
      - ./config/wazuh_indexer/wazuh.indexer.yml:/usr/share/wazuh-indexer/config/opensearch.yml
      - ./config/wazuh_indexer/internal_users.yml:/usr/share/wazuh-indexer/plugins/opensearch-security/securityconfig/internal_users.yml

  wazuh.dashboard:
    image: wazuh/wazuh-dashboard:4.3.10
    hostname: wazuh.dashboard
    restart: always
    ports:
      - 5601:5601
    environment:
      - INDEXER_USERNAME=admin
      - INDEXER_PASSWORD=SecretPassword
      - WAZUH_API_URL=https://wazuh.manager
      - API_USERNAME=wazuh-wui
      - API_PASSWORD=MyS3cr37P450r.*-
    volumes:
      - ./config/wazuh_indexer_ssl_certs/wazuh.dashboard.pem:/usr/share/wazuh-dashboard/certs/wazuh-dashboard.pem
      - ./config/wazuh_indexer_ssl_certs/wazuh.dashboard-key.pem:/usr/share/wazuh-dashboard/certs/wazuh-dashboard-key.pem
      - ./config/wazuh_indexer_ssl_certs/root-ca.pem:/usr/share/wazuh-dashboard/certs/root-ca.pem
      - ./config/wazuh_dashboard/opensearch_dashboards.yml:/usr/share/wazuh-dashboard/config/opensearch_dashboards.yml
      - ./config/wazuh_dashboard/wazuh.yml:/usr/share/wazuh-dashboard/data/wazuh/config/wazuh.yml
    depends_on:
      - wazuh.indexer
    links:
      - wazuh.indexer:wazuh.indexer
      - wazuh.manager:wazuh.manager

volumes:
  wazuh_api_configuration:
  wazuh_etc:
  wazuh_logs:
  wazuh_queue:
  wazuh_var_multigroups:
  wazuh_integrations:
  wazuh_active_response:
  wazuh_agentless:
  wazuh_wodles:
  filebeat_etc:
  filebeat_var:
  wazuh-indexer-data:

```

**Comment by teddytpc1** [comment]
Hi @jeevadotnet.
I was looking for the `ResizeObserver` error and found [this](https://github.com/wazuh/wazuh-kibana-app/issues/4289#issuecomment-1167212777).

1. Would you tell me what browser and version are you using? 
2. Can you try updating the browser?
3. Can you try with another browser (in its latest version)?

**Comment by jeevadotnet** [comment]
Hi @teddytpc1 

Thanks fro the feedback. I use `Basilisk 2022.11.04 (64-bit)` which is a Firefox derivative, which I configured for SOCKS5 proxies. 

I tried now with Firefox 102.5.0esr (64-bit) for Kali and it seems to be working correctly
![image](https://user-images.githubusercontent.com/12102129/206216100-41a09dc0-c991-4b25-9360-7dc918aa1731.png)

Thanks for the pointer. It has been quite some time since I had to change a browser to fix an issue. Now I can finally deploy the software :) Thank you
